packages/connect-voting-disputable/src/models/Setting.ts

```diff
@@ -28,4 +26,12 @@ export default class Setting {
     this.quietEndingExtension = data.quietEndingExtension
     this.createdAt = data.createdAt
   }
+
+  get formattedSupportRequiredPct(): string {
+    return formatBn(this.supportRequiredPct, PCT_DECIMALS)
```

Comment:
If we decided to use `PCT_DECIMALS` as default we can remove it in a few places. Or do you think is better to be explicit about it?

Reply:
Well, actually note that we are using different decimals for these operations, 16 for percentages and 18 for token amounts, then I'd keep it